modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+You can upgrade the platform version of a managed {sno} cluster with the {lcao}.
+Using a generated seed image allows you to skip up to two minor releases and it is faster than the standard upgrade method.
```

Comment:
I would start with some context, saying this operator brings an alternative way of managing upgrades on SNO. Alternative to the standard upgrades method (with a link). And I suggest point first to the main features, it does it much more faster to satisfy usual upgrade requirements on Telco. And also, that you can do OCP upgrades x.y+2 seamless. In the first paragraph maybe is not needed to talk about seed images.
Keeping the things about seed images, that are implementation details, for the second paragraph.